code_testing.ts

```diff
@@ -67,11 +66,31 @@ function delay(ms: number)
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+async function test_connection():Promise<Boolean>{
+    return new Promise<Boolean>((resolve,reject)=>{
+        db.getAllIndividuals().then(result=>{
```

Comment:
using this wont scale well, though it will work for now. You can actually just use sequelize for this
https://sequelize.org/master/manual/getting-started.html
```js
sequelize.authenticate().then(() => {
  console.log('Connection has been established successfully.');
}.catch((error) => {
  console.error('Unable to connect to the database:', error);
})
```